src/Maker/MakeCrud.php

```diff
@@ -179,7 +180,8 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
                     'entity_var_singular' => $entityVarSingular,
                     'entity_twig_var_singular' => $entityTwigVarSingular,
                     'entity_identifier' => $entityDoctrineDetails->getIdentifier(),
-                    'use_render_form' => method_exists(AbstractController::class, 'renderForm'),
+                    // @legacy - Remove when support for Symfony <6 is dropped
+                    'use_render_form' => Kernel::VERSION_ID < 60000,
```

Comment:
I believe it should be:

```suggestion
                    'use_render_form' => Kernel::VERSION_ID < 62000,
```

MakerBundle supports 5.4 and higher only, and `renderForm()` was added in 5.3. So:

* 5.4 - 6.1, we should use `renderForm()`
* In 6.2 & higher, use `render()`, always without `createView()`